File: Cientista/cientista.c
#include "cientista.h"

void iniciarCientista(Cientista *cientista) {

    cientista->vidaMaxima = 60;
    cientista->vida = cientista->vidaMaxima;

    cientista->forca = 10;
    cientista->resistencia = 5;
    cientista->precisao = 15;
    cientista->evasao = 10;

    cientista->chanceEnvenenamento = 30;
    cientista->chanceQueimadura = 30;

    cientista->chanceFrascoSangue = 10;
    cientista->chancePocaoCura = 60;
    cientista->chanceAntidoto = 60;
}

File: personagens/Cientista.java
package personagens;

public class Cientista {
    public int vida;
    public int vidaMaxima;
    public int forca;
    public int resistencia;
    public int precisao;
    public int evasao;

    public int sangramento;
    public int turnosSangramento;

    public int chanceEnvenenamento;
    public int chanceQueimadura;
    public int chanceFrascoSangue;
    public int chancePocaoCura;
    public int chanceAntidoto;

    public Cientista() {
        vidaMaxima = 160;
        vida = vidaMaxima;
        forca = 18;
        resistencia = 10;
        precisao = 15;
        evasao = 10;

        sangramento = 0;
        turnosSangramento = 0;

        chanceEnvenenamento = 30;
        chanceQueimadura = 30;
        chanceFrascoSangue = 10;
        chancePocaoCura = 60;
        chanceAntidoto = 60;
    }
}
